sig-contributor-experience/contribex-review-guidelines-on-test-infra.md

```diff
@@ -0,0 +1,123 @@
+
+- trigger
+- owners
+- approve
+- help
+- size
+- label
+- blockades
+- slack
+- welcome
+- require_matching_label
+
+#### Plugins Configured for Specific Orgs/Repos
+
+**Organizations:**
+- kubernetes
+- kubernetes-client
+- kubernetes-sigs
+- etcd-io
+
+**Repositories:**
+- kubernetes/kubernetes
+- kubernetes/community
+- kubernetes/org
+- kubernetes/publishing-bot
+- kubernetes-sigs/contributor-playground
+
+#### External Plugins Configured for Specific Orgs/Repos
+
+**Organizations:**
+- kubernetes
+- kubernetes-sigs
+- kubernetes-csi
+- kubernetes-client
+
+---
+
+### Changes to [`config/prow/config.yaml`](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/config.yaml)
+
+- [slack_reporter_configs](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/config.yaml#L215-L226)
+- [branch-protection](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/config.yaml#L228-L636)
+- [tide](https://github.com/kubernetes/test-infra/blob/master/config/prow/config.yaml#L638-L844)
+- [github_reporter](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/config.yaml#L850-L853)
+
+---
+
+### Changes to [`config/prow/OWNERS`](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/OWNERS#L1) file
+
+Any changes to the reviewers/approvers or filters listed in the `config/prow/OWNERS` file.
+
+---
+
+### Changes to Prow CI Jobs and Their OWNERS Files
+
+Changes to the definition of the following Prow CI jobs files as well as their respective OWNERS files, owned directly or indirectly by SIG ContribEx:
+
+- All files under [kubernetes/community/](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes/community) path
+- All files under [kubernetes/org/](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes/org) path
+- [kubernetes/sig-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml)
+- [kubernetes/sig-k8s-infra/trusted/sig-contribex-peribolos.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-peribolos.yaml)
+- [kubernetes/sig-k8s-infra/trusted/sig-k8s-infra-groups.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-k8s-infra/trusted/sig-k8s-infra-groups.yaml)
+- [config/jobs/kubernetes-sigs/slack-infra/slack-infra-config.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/slack-infra/slack-infra-config.yaml)
+
+---
+
+### Changes to `Label_sync` Path
+
+Any changes to the files under [`label_sync`](https://github.com/kubernetes/test-infra/tree/master/label_sync) path.
+
+---
+
+## Out of Scope
```

Comment:
Do we want to also link https://github.com/kubernetes/community/blob/master/sig-contributor-experience/charter.md#out-of-scope here? To maybe xref?